docker/images/pinot/etc/jmx_prometheus_javaagent/configs/server.yml

```diff
@@ -85,6 +77,14 @@ rules:
 - pattern: "\"org.apache.pinot.common.metrics\"<type=\"ServerMetrics\", name=\"pinot.server.realtimeConsumptionExceptions\"><>(\\w+)"
   name: "pinot_server_realtime_consumptionExceptions_$1"
   cache: true
+- pattern: "\"org.apache.pinot.common.metrics\"<type=\"ServerMetrics\", name=\"pinot.server.([^\\.]*?)_(OFFLINE|REALTIME)\\-(.+)\\-(\\w+).(invalidRealtimeRowsDropped|incompleteRealtimeRowsConsumed|rowsWithErrors|realtimeRowsConsumed)\"><>(\\w+)"
+  name: "pinot_server_$4_$2"
+  cache: true
+  labels:
+    table: "$1"
+    tableType: "$2"
+    topic: "$3"
+    partition: "$5"
```

Comment:
The partition label seems incorrect here. When I run your code I get the following 

```
Input string 'pinot.server.meetupRsvp_OFFLINE-meetupRSVPEvents-1.realtimeRowsConsumed' matches the regex pattern: pinot.server.([^\.]*?)_(OFFLINE|REALTIME)\-(.+)\-(\w+).(invalidRealtimeRowsDropped|incompleteRealtimeRowsConsumed|rowsWithErrors|realtimeRowsConsumed)
metricName: pinot.server.meetupRsvp_OFFLINE-meetupRSVPEvents-1.realtimeRowsConsumed
Group 0: pinot.server.meetupRsvp_OFFLINE-meetupRSVPEvents-1.realtimeRowsConsumed
Group 1: meetupRsvp
Group 2: OFFLINE
Group 3: meetupRSVPEvents
Group 4: 1
Group 5: realtimeRowsConsumed
```

So it should be $4 and not $5, right?
And metric name should contain $5 instead of $4

Reply:
Changed as discussed